themes/bootstrap3/templates/myresearch/profile.phtml

```diff
@@ -84,6 +84,46 @@
     <?php endif; ?>
   </div>
 
+  <?php $user = $this->auth()->isLoggedIn(); ?>
+  <?php if ($this->auth()->getManager()->supportsPersistentLogin($user)): ?>
+    <div class="myresearch-profile-header">
+      <h3><?=$this->transEsc('Saved logins')?></h3>
+    </div>
+    <?php $tokens = $user->getLoginTokens($user->id); ?>
+    <?php if (count($tokens) === 0): ?>
+      <div class="my-profile-col"><?=$this->transEsc('No saved logins')?></div>
+    <?php else: ?>
+      <table class="login-tokens-table table table-striped">
```

Comment:
Should this table be given responsive styling similar to what was recently added in #3308?

Reply:
Table is now responsive